### PR TITLE
fix(docs): better highlight macos link, remove bazelisk recommendation

### PR DESCRIPTION
**Pull request by jaysachs**

### CODE REVIEW

**Review by salguarnieri — APPROVED** [review]

File: kythe/web/site/getting-started-macos.md [hunk 1]
```diff
@@ -63,19 +63,6 @@ for pkg in asciidoc bison brotli flex go graphviz leveldb node openjdk parallel
    brew install $pkg
 done
 
-# For Bazel, we recommend using Bazelisk, a wrapper tool that uses
```
Comment (creachadair) [inline kythe/web/site/getting-started-macos.md:66]: This leaves macOS users without clear instructions for how to install Bazel. What's the current recommendation, if not to use Bazelisk?

Comment (jaysachs) [reply kythe/web/site/getting-started-macos.md:66]: Oh, good catch.

Comment (jaysachs) [reply kythe/web/site/getting-started-macos.md:66]: Wait ... that is on the main "getting started". The "getting started for macOS" is about dependencies. Bazel install instructions are general ("surf to bazel.io and install")

Comment (creachadair) [reply kythe/web/site/getting-started-macos.md:66]: Fair enough. That didn't used to work. Interestingly, one of their four recommended ways to install Bazel on macOS is via Bazelisk.

(I don't object to omitting the instructions, but I did have to drill about four pages deep into the mess of bazel.io to find that)

**Pull request merged**